src/pages/verify/e-ids/norwegian-bankid.mdx

```diff
@@ -49,11 +49,12 @@ Two types of Norwegian BankID are available:
 
 Test users are created through the web page at [https://ra-preprod.bankidnorge.no/#/search/endUser](https://ra-preprod.bankidnorge.no/#/search/endUser).
 
-1. Go to the "TEST NUMBER GENERATOR" to generate a random, valid SSN
+1. Go to the ["TEST NUMBER GENERATOR"](https://ra-preprod.bankidnorge.no/#/generate) to generate a random, valid SSN
 2. It now says "Could not find any bankIDs for ..."
 3. Fill out the first name, last name, and BankID friendly name
 4. Click "Order" to initiate the process
-5. Once the process is complete, you will have a test user. User name is the generated SSN, one time password (OTP) is always "otp", and password is always "qwer1234"
+5. Click the pencil icon and add a phone number and an email that you want to associate with the test user
```

Comment:
Is a valid phone number required (and must it be Norwegian in that case), or can one just supply anything ?

Reply:
It can now work with any country code, but I believe the number should be valid

Reply:
Does "valid" mean "a specific number of digits" or something else ?

Reply:
Disregard my previous comment, the number doesn't have to be valid (active/functional)
It only has to adhere to the phone number format (differs a bit per country in the number of digits. For instance, for DK this will be accepted: 45 71 11 11 11

Reply:
OK, then perhaps we should add something like "any number with the correct number of digits for a given country code should work" ?

Reply:
Nvm, you already added something better :)

Reply:
I made it a bit more clear now